change/@ni-nimble-components-8010f392-806a-4399-bf80-9fcd41f97cb5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note that since the default size changed, the Chromatic tests in the build status page are showing some diffs. You can either:
1. approve those diffs using the chromatic UI (let me know if you need help/access)
2. set a default size in the storybook stories so that there is no longer a diff (or perhaps just some minor pixel diffs)

I don't have a strong preference but one benefit of 2 is that it makes for smaller image snapshots, which makes it less likely that we'll run into Chromatics image size limitations.

Reply:
applied ... I did implement a default size (300x300), as suggested by Milan, for the web-component, inside the :host style.